README.CentOS6.md

```diff
@@ -14,39 +14,97 @@ For further details on the install process, please see the LORIS GitHub Wiki Cen
 
 Default dependencies installed by CentOS 6.x may not meet the version requirements LORIS deployment or development.
 * MySQL 5.7 is supported for LORIS 17.*
-* PHP 7 is supported for LORIS 17.* - upgrade your PHP manually
+* PHP 7 is supported for LORIS 17.* - see below to upgrade your PHP manually
 
 The yum packages to be installed vary from any Ubuntu packages referenced in the LORIS README.
 
 The following should be installed with yum:
- 
-```bash
-yum install httpd
-yum install php
-yum install php-pdo
-yum install php-mysql
-yum install mysql
+ * Apache2
+ * PHP
+ * MySQL
+ * PHP Composer
+
+**Apache2:**
+```
+sudo yum install httpd 
+sudo service httpd start
+```
+**PHP:**
+```
+sudo yum install php php-pdo php-mysql 
 ```
 
-PHP Composer must also be installed:
-```bash
-curl -sS https://getcomposer.org/installer | php
+*NOTE:* As of Loris 18.0 php7 is required (but not yet officially supported by CentOS). To upgrade php follow the following instructions
```

Comment:
As of LORIS 17.0, not 18.0 PHP7 was the minimum required version..

(There should also be a ":" after the word instructions.)